src/plugin.js

```diff
@@ -1,8 +1,29 @@
-import 'webvr-polyfill/build/webvr-polyfill';
+import window from 'global/window';
+
+// this is used to configure webvr-polyfill
+// see https://github.com/googlevr/webvr-polyfill#configuration
+const WebVRConfig = {
```

Comment:
all import statements are hoisted to the top. We should also write our code that way.